cmd/skywire-cli/commands/node/gen-config.go

```
@@ -186,3 +189,19 @@ func defaultSkyproxyClientConfig() visor.AppConfig {
 		Port:      routing.Port(skyenv.SkyproxyClientPort),
 	}
 }
+
+func getLocalIPAddress() string {
+	addrs, err := net.InterfaceAddrs()
+	if err != nil {
+		log.WithError(err)
```

I would return an error here and not continue function execution in that case.